lib/teamsnap.rb

```diff
@@ -34,6 +35,15 @@ def init(opts = {})
       ##   create universally accessible TeamSnap.root_client
       self.root_client = TeamSnap::Client.new(:token => token)
 
+      ##   include any feature headers
+      if opts[:header]
+        if opts.fetch(:header) == "ghost_contact"
+          self.root_client.headers = self.root_client.headers.merge(
+          {"X-Teamsnap-Api-Features" => "ghost_contact"}
```

Comment:
same question as readme about exposing ghost. What about passing in the entire thing?

```
{"X-Teamsnap-Api-Features" => "ghost_contact"}
```